tgui/src/interfaces/cryo.ract

```diff
@@ -1,53 +0,0 @@
-<ui-display title='Occupant'>
```

Comment:
don't delete the old ract file, we're keeping them around until old tgui is deprecated for reference and backup purposes.